webapp/app/partials/main/jobs.html

```diff
@@ -51,17 +51,17 @@
     <div class="result-set-body" th-clone-jobs ></div>
 </div>
 
-<div ng-show="result_sets.length == 0 && !isLoadingRsBatch.appending &&
+<div ng-show="result_sets.length != 0 && !isLoadingRsBatch.appending &&
```

Comment:
We don't need to change the operator here - the problematic behaviour is due to our identification of a valid vs invalid repo - which is done using |&& currentRepo.url| (see below).